src/main/java/com/promise8/wwwbe/model/dto/MeetingMainGetResDtoWrapper.java

```diff
@@ -0,0 +1,127 @@
+                            confirmedPromise.promiseTime,
+                            confirmedPromise.promisePlace,
+                            confirmedPromise.hostAndVotingCnt.hostName
+                    ));
+                } else {
+                    meetingMainIngGetResDtoList.add(MeetingMainGetResDto.of(
+                            meeting,
+                            meeting.getMeetingUserEntityList(),
+                            confirmedPromise.hostAndVotingCnt.votingUserCount,
+                            confirmedPromise.promiseDate,
+                            confirmedPromise.promiseTime,
+                            confirmedPromise.promisePlace,
+                            confirmedPromise.hostAndVotingCnt.hostName
+                    ));
+                }
+            } else {
+                HostAndVotingCnt hostAndVotingCnt = getHostAndVotingCnt(meeting.getMeetingUserEntityList(), meeting.getUserEntity().getUserId());
+                meetingMainIngGetResDtoList.add(MeetingMainGetResDto.of(
+                        meeting,
+                        meeting.getMeetingUserEntityList(),
+                        hostAndVotingCnt.votingUserCount,
+                        null,
+                        null,
+                        null,
+                        hostAndVotingCnt.hostName
+                ));
+            }
+        }
+
+        return MeetingMainGetResDtoWrapper.builder()
+                .meetingMainIngGetResDtoList(meetingMainIngGetResDtoList)
+                .meetingMainEndGetResDtoList(meetingMainEndGetResDtoList)
+                .build();
+    }
+
+    private static ConfirmedPromise getConfirmedPromise(List<MeetingUserEntity> meetingUserEntityList, Long hostId) {
+        ConfirmedPromise confirmedPromise = new ConfirmedPromise();
+        for (MeetingUserEntity meetingUser : meetingUserEntityList) {
+            if (confirmedPromise.promiseDate == null) {
+                for (MeetingUserTimetableEntity meetingUserTimetable : meetingUser.getMeetingUserTimetableEntityList()) {
+                    if (meetingUserTimetable.getIsConfirmed()) {
+                        confirmedPromise.promiseDate = meetingUserTimetable.getPromiseDate();
+                        confirmedPromise.promiseTime = meetingUserTimetable.getPromiseTime();
+                        break;
+                    }
+                }
+            }
+
+            if (confirmedPromise.promisePlace == null) {
+                for (MeetingPlaceEntity meetingPlace : meetingUser.getMeetingPlaceEntityList()) {
+                    if (meetingPlace.getIsConfirmed()) {
+                        confirmedPromise.promisePlace = meetingPlace.getPromisePlace();
+                        break;
+                    }
+                }
+            }
+
+            confirmedPromise.hostAndVotingCnt.votingUserCount += meetingUser.getPlaceVoteEntityList() != null && meetingUser.getPlaceVoteEntityList().size() > 0 ? 1 : 0;
```

Comment:
```suggestion
            confirmedPromise.hostAndVotingCnt.votingUserCount += isCondition(meetingUser) ? 1 : 0;
```

로직이 긴데 삼항, 연산문까지 섞여서 가독성이 떨어지는 것 같습니닷. 조건식은 따로 메소드로 빼서 단순화하는게 좋을 듯합니당

+ 콜렉션은 null체크보단 CollectionUtils.isEmpty() 사용하면 더 좋을듯

Reply:
네 확인했습니다~

Reply:
수정 완료요!